src/k8s_helpers.rs

```diff
@@ -49,3 +55,67 @@ pub fn create_selector(key: &str, value: &str) -> BTreeMap<String, String> {
     btree.insert(key.to_string(), value.to_string());
     btree
 }
+
+#[allow(clippy::too_many_arguments)]
+pub fn create_replica_set(
```

Comment:
I have no idea if this works, so I'll assume it's all correct :sweat_smile:

Reply:
haha it does. but i could change `create_replica_set()` to accept a struct instead of all the args.